modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsListTransitionTasks.testcase

```diff
@@ -0,0 +1,166 @@
+	@ignore = "Test Stub"
+	@priority = "2"
+	test MessageIsDisplayedWhenConnectionIsLost {
+
+		// TODO LRQA-67823 MessageIsDisplayedWhenConnectionIsLost pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the message (The selected steps have transitioned successfully) is displayed when several steps are transitioned at once"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test MessageIsDisplayedWhenSeveralStepsAreTransitionedAtOnce {
+
+		// TODO LRQA-67820 MessageIsDisplayedWhenSeveralStepsAreTransitionedAtOnce pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the message (Select a transition) and (All steps require a transition to be selected to complete this action.) are displayed when the user clicks on Done without selecting all transitions"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test MessageIsDisplayedWhenSomeTransitionIsNotSelected {
+
+		// TODO LRQA-67818 MessageIsDisplayedWhenSomeTransitionIsNotSelected pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that, in (Choose Transition Per Step) modal, the user can add comments when transitioning tasks"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test OnChooseTransitionPerStepCanAddCommentToATransitionTask {
+
+		// TODO LRQA-67812 OnChooseTransitionPerStepCanAddCommentToATransitionTask pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that users can filter the tasks to be transitioned by assignees"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test OnChooseTransitionPerStepCanFilterByAssignee {
+
+		// TODO LRQA-67803 OnChooseTransitionPerStepCanFilterByAssignee pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that users can filter the tasks to be transitioned by steps"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test OnChooseTransitionPerStepCanFilterByStep {
+
+		// TODO LRQA-67805 OnChooseTransitionPerStepCanFilterByStep pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that only the tasks selected on (Select Steps to Transition) are being considered to be transitioned in the (Choose Transition Per Step) step"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test OnlySelectedTasksAreConsideredToBeTransitioned {
+
+		// TODO LRQA-67811 OnlySelectedTasksAreConsideredToBeTransitioned pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that when there are 11 or more instances to be transitioned with the same step, only the first 10 instances are displayed with the (Show All/Show Less) buttons available to be selected"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test ShowAllLessButtonAvailable {
+
+		// TODO LRQA-67802 ShowAllLessButtonAvailable pending implementation
+
+	}
+
+	@description = "LPS-108138 - Verify that the tasks on (Choose Transition Per Step) are grouped by versioned tasks (when there are changes in the workflow process)"
```

Comment:
Please, remove the parenthesis from (Choose Transition Per Step).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.